base/math.jl

```diff
@@ -629,7 +629,15 @@ julia> hypot(3, 4im)
 hypot(x::Number, y::Number) = hypot(promote(x, y)...)
 hypot(x::Complex, y::Complex) = hypot(abs(x), abs(y))
 hypot(x::T, y::T) where {T<:Real} = hypot(float(x), float(y))
-hypot(x::T, y::T) where {T<:Number} = (z = y/x; abs(x) * sqrt(one(z) + z*z))
+function hypot(x::T, y::T) where {T<:Number}
+    #Return Inf if either or both imputs is Inf (Compliance with IEEE754)
```

Comment:
```suggestion
    # Return Inf if either or both inputs is Inf (Compliance with IEEE754)
```

Reply:
The is also another identical typo below.